### PR TITLE
Fix jackson and resteasy deps

### PR DESCRIPTION
**Pull request by andreaturli**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -91,14 +91,16 @@
...
         <slf4j.version>1.6.6</slf4j.version>  <!-- used for java.util.logging jul-to-slf4j interception -->
         <guava.version>17.0</guava.version>
-        <xstream.version>1.4.7</xstream.version>
-        <fasterxml.jackson.version>2.7.0</fasterxml.jackson.version>  <!-- more recent jackson, but not compatible with old annotations! -->
-        <jersey.version>1.19</jersey.version>
+	      <xstream.version>1.4.7</xstream.version>
+	      <!-- double-check downstream projects before changing jackson and resteasy versions -->
+        <fasterxml.jackson.version>2.4.5</fasterxml.jackson.version>
+	      <resteasy.version>3.0.8.Final</resteasy.version>
+	      <jersey.version>1.19</jersey.version>
```
Comment (sjcorbett) [inline pom.xml]: Weird indentation

**Comment by sjcorbett** [comment]
@andreaturli Thanks. Will wait for Jenkins and merge.

**Comment by andreaturli** [comment]
@sjcorbett, does the [failure](https://builds.apache.org/job/brooklyn-server-pull-requests/org.apache.brooklyn$brooklyn-core/70/testReport/junit/org.apache.brooklyn.entity.stock/BasicStartableTest/testTransitionsThroughLifecycles/) looks related to you?

**Comment by andreaturli** [comment]
close and re-open to rebuild

**Comment by sjcorbett** [comment]
No, looks like another unreliable test.

**Pull request merged**